app/helpers/money_helper.rb

```diff
@@ -0,0 +1,17 @@
+module MoneyHelper
+  include MoneyRails::ActionViewExtension
+
+  Money.default_bank.add_rate(:USD, :EUR, 0.88)
```

Comment:
Should we make this not static? Is there a standard way, e.g. Stripe?

Reply:
I think so—I'll add a TODO comment. There's a few different [backends available](https://github.com/RubyMoney/money#implementations) for live conversions, I'll take a look at them for the next PR. This seems adequate for now for the figures reported by `Counts` though.

Reply:
Sounds good. I'm all for not blocking on this, just curious if there was a good way. From a cursory look at [Stripe's Currency Conversion](https://stripe.com/docs/currencies/conversions) it seems like they aren't the solution.

I think we can punt on this for now, and deal with it once we're finished language things.

Reply:
Added an issue to track this #952